components/ILIAS/FileDelivery/src/Services.php

```diff
@@ -83,6 +83,7 @@ protected function getBaseURI(): string
         return $this->base_uri ?? $this->base_uri = rtrim(
             $this->http->request()->getUri()->getScheme()
             . '://' . $this->http->request()->getUri()->getHost()
+            . ($this->http->request()->getUri()->getPort() ? ':' . $this->http->request()->getUri()->getPort() : '')
```

Comment:
Isn't this the same as in https://github.com/ILIAS-eLearning/ILIAS/pull/9558 ? you may rebase your branch and this should vanish IMO.